extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyDotNames.java

```diff
@@ -95,4 +99,8 @@ public boolean test(DotName name) {
             // Vert-x
             DotName.createSimple("io.vertx.core.json.JsonArray"),
             DotName.createSimple("io.vertx.core.json.JsonObject")));
+    
+    private static final String[] PACKAGES_IGNORED_FOR_REFLECTION = {
+            "javax.json"
```

Comment:
To be totally safe, I think you need to add a dot at the end of the package name. It's not a biggie for this one but might be in the future for other packages.

It's probably better to do it here than in the startsWith as it will avoid creating new strings.

Reply:
Ok. Let's be "totally safe" :-)